README.md

```diff
@@ -94,6 +96,24 @@ You now specify your action space in the json file you pass in through
 `MODEL_METADATA_FILE_S3_KEY`, which is defaulted to
 `bucket/custom_files/model_metadata.json`
 
+### Altering hyperparameters
+To override default hyperparameters, in `rl_deepracer_coach_robomaker.py` uncomment the fields that you wish to override and set their value.
+
+Default values can be found in `sagemaker_graph_manager.py`
+
+### Using a pretrained model
+To train based on an existing model, copy it into `rl-deepracer-pretrained` in bucket and uncomment options
+`pretrained_s3_bucket` and `pretrained_s3_prefix` in `rl_deepracer_coach_robomaker.py`
+
+### dr_util.py
```

Comment:
Can you add a warning of not taking any responsibility of what this does to AWS accounts? I know that goes for anything you don't write yourself, but because this is directly interacting with an AWS account, I think it needs an extra warning.

Reply:
Added here and in script

Reply:
Looks good, cheers!